element.go

```diff
@@ -0,0 +1,62 @@
+package pokemonbattlelib
+
+import "fmt" 
+type ElementalType uint32
+var effect int
+
+const (
+	Normal ElementalType = 65536 >> iota
+	Fight //66048 100000000000000000
+	Flying  
+	Poison  
+	Ground  
+	Rock  
+	Bug
+	Ghost
+	Steel
+	Fire
+	Water
+	Grass
+	Electric
+	Psychic
+	Ice
+	Dragon
+	Dark
+)
+
+func contains(arr []uint32, str uint32) bool {
+   for _, a := range arr {
+      if a == str {
+         return true
+      }
+   }
+   return false
+}
+ 
+func getEffect(move, def ElementalType) int {
+    var match =uint32(move| def)
+    fmt.Println("match spot: ", uint32(move| def))
+    fmt.Printf("%b", int32(132096))
+    noEffect := []uint32{132096,66560,16896,40960,133120,8208,9}
```

Comment:
You can enter binary literal numbers by starting the number with `0b`. For example: `0b101 == 5`

Reply:
Thank you. They are easier to read